pages/nft/iscn/_iscnId.vue

```diff
@@ -387,19 +398,28 @@ export default class NFTTestMintPage extends Vue {
         memo,
       }
     } catch (err) {
+      logTrackerEvent(this, 'IscnMintNFT', 'CheckArweaveIdExistsAndEstimateFeeError', (err as Error).toString(), 1);
       // eslint-disable-next-line no-console
       console.error(err)
       throw new Error('CANNOT_ESTIMATE_ARWEAVE_FEE')
     }
   }
 
   async sendArweaveFeeTx({ to, amount, memo }: { to: string, amount: BigNumber, memo: string }): Promise<void> {
-    if (!this.signer) throw new Error('SIGNER_NOT_INITED')
-    if (!to) throw new Error('TARGET_ADDRESS_NOT_SET')
+    if (!this.signer) {
+      logTrackerEvent(this, 'IscnMintNFT', 'SendArweaveFeeTxSignerError', 'SIGNER_NOT_INITED', 1);
+      throw new Error('SIGNER_NOT_INITED')
+    }
+    if (!to) {
+      logTrackerEvent(this, 'IscnMintNFT', 'SendArweaveFeeTxTargetError', 'TARGET_ADDRESS_NOT_SET', 1);
+      throw new Error('TARGET_ADDRESS_NOT_SET')
+    }
```

Comment:
We can set all log action as `SendArweaveFeeTxError` and use different label for each cases